content/doc/book/pipeline/jenkinsfile.adoc

```
@@ -839,33 +668,24 @@ node {
 
 ===== Combining credentials in one step
 
-Using the *Snippet Generator*, you can make multiple credentials available
-within a single `withCredentials( ... ) { ... }` step by doing the following:
+Using the *Snippet Generator*, you can make multiple credentials available within a single `withCredentials( ... ) { ... }` step by doing the following:
 
-. From the Jenkins home page (i.e. the Dashboard of Jenkins' classic UI), click
-  the name of your Pipeline project/item.
-. On the left, click *Pipeline Syntax* and ensure that the *Snippet Generator*
-  link is in bold at the top-left. (If not, click its link.)
-. From the *Sample Step* field, choose *withCredentials: Bind credentials to
-  variables*.
+. From the Jenkins home page (i.e. the Dashboard of Jenkins' classic UI), click the name of your Pipeline project/item.
+. On the left, click *Pipeline Syntax* and ensure that the *Snippet Generator* link is in bold at the top-left. (If not, click its link.)
+. From the *Sample Step* field, choose *withCredentials: Bind credentials to variables*.
 . Click *Add* under *Bindings*.
-. Choose the credential type to add to the `withCredentials( ... ) { ... }` step
-  from the dropdown list.
-. Specify the credential *Bindings* details. Read more above these in the
-  procedure under <<#for-other-credential-types,For other credential types>>
-  (above).
-. Repeat from "Click *Add* ..." (above) for each (set of) credential/s to add to
-  the `withCredentials( ... ) { ... }` step.
-. Click *Generate Pipeline Script* to generate the final
-  `withCredentials( ... ) { ... }` step snippet.
+. Choose the credential type to add to the `withCredentials( ... ) { ... }` step from the dropdown list.
+. Specify the credential *Bindings* details.
+  Read more above these in the procedure under <<#for-other-credential-types,For other credential types>> (above).
+. Repeat from "Click *Add* ..." (above) for each (set of) credential/s to add to the `withCredentials( ... ) { ... }` step.
+. Click *Generate Pipeline Script* to generate the final `withCredentials( ... ) { ... }` step snippet.
```

```suggestion
. Select *Generate Pipeline Script* to generate the final `withCredentials( ... ) { ... }` step snippet.
```
Suggestion to replace `Click` with `Select`.